src/lib.rs

```diff
@@ -2,9 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+mod errors;
```

Comment:
Note that with this module being private, users of the crate may be limited on what they can do with the errors returned by `SignatureShare`